citations-tool/tool/src/java/org/sakaiproject/citation/tool/CitationHelperAction.java

```diff
@@ -614,12 +605,16 @@ public static void clearAdvancedFormState( SessionState state )
 	protected static final String PROP_USE_RELEASE_DATE = "useReleaseDate";
 	protected static final String PROP_USE_RETRACT_DATE = "useRetractDate";
 
+	/** Property for long open introduction (user settable). [String] */
+	static final String PROP_INTRODUCTION = "CHEF:introduction";
```

Comment:
Discussed in office, move to API and from the CHEF prefix (switch to something pointing at citations).